### PR TITLE
Mexehamy patch 1

### PR DESCRIPTION
**Pull request by mexehamy**
@voroninman, PR is ready for CR

### CODE REVIEW

**Comment by voroninman** [comment]
Please provide a useful title for the PR.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ Before writing this blog I haven't had any knowledges in python and especially i
 #How to start
 
 ```
+
+cd ~/blog2
+virtualenv env
+source /env/bin/activate
```
Comment (voroninman) [inline README.md]: I guess there is a typo and we need the path without leading `/`.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,18 @@ Before writing this blog I haven't had any knowledges in python and especially i
...
+pip install django
+pip install django-disqus
+
+```
+
+#How to run server
+
+```
 cd ~/blog2
 virtualenv env
```
Comment (voroninman) [inline README.md]: Why do we need to create a new virtual environment each time we run our blog?
Also do we need to activate the environment?

File: README.md (hunk 1 above)
Comment (mexehamy) [inline README.md]: yes, exactly

**Comment by voroninman** [comment]
@mexehamy Good improvment! :+1: Still has some issue when we just run. I left a few comments but overall LGTM. :metal:

File: README.md (hunk 2 above)
Comment (mexehamy) [inline README.md]: Right, we don't have to create and activate it each time. I'll delete this line

**Pull request merged**